pocs/observatory.py

```diff
@@ -379,6 +380,14 @@ def get_standard_headers(self, observation=None):
         # Add observation metadata
         headers.update(observation.status())
 
+        # Explicitly convert EQUINOX to float
+        try:
+            equinox = float(headers['equinox'].replace('J', ''))
+        except BaseException:
+            equinox = 2000.  # We assume J2000
+        finally:
```

Comment:
Nit: Doesn't need to be a finally clause.

Reply:
I was mostly trying to make that a contained block. Can remove.